caffe2/operators/fully_connected_op.cc

```diff
@@ -154,7 +154,8 @@ OPERATOR_SCHEMA(FCTransposed)
     .SetDoc(R"DOC(
 Same as FC, but weight matrix is supposed to be already pretransposed.
 FCTransposed stands for calling blass with no noTrans, noTrans
-)DOC");
+)DOC")
+    .InheritOnnxSchema("FCTransposed");
```

Comment:
Maybe we can add an overload for `InheritOnnxSchema` that doesn't need to pass the same caffe2 schema name:
```c++
OpSchema& OpSchema::InheritOnnxSchema() {
  return InheritOnnxSchema(type_);
}
```

Reply:
Cool! I would like to do it in a separate diff as it will touch `core` and many other files.

Reply:
sure